### DISCUSSION TITLE
How do you splice a corpus object into string vector of length 1? as.character %> tokenize %>% % unlist?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#147] [Q&A] [answered]
Let's say I have a corpus object called `x_corp`
I need to splice it into multiple string vector of length 1 so I can feed them into a katadasar function for stemming purpose. 
How do I do that?

I've tried this things bellow: 

```
kata <- function(x){
  y <- lapply(x[]$content, as.character)
  z <- y %>% tokenize_word_stems()
  lapply(unlist(z), katadasar)
  }

eks_korp <- tm_map(corpus_tweet, kata)
```

But the function kept on running endlessly, never found peace, so I always had to end its course before I could find out whether my attempt was wrong or not right. Help? Thanks.

**Comment by NabiilahArdini** [comment, marked as answer]
Dear Mr. @Hertzspr,
I'm afraid, I still haven't understood your question clearly, but perhaps this article could help. In here, the author uses different approach/packages for text cleaning though: https://algotech.netlify.app/blog/text-cleaning-bahasa/

**Reply by Hertzspr** [reply]
In the end, I used tidytext approach rather than corpus based approach.
